01week/datatypes.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+let num1 = 1;
+let num2 = 2;
+const string1 = '1';
+const dataType = 3;
+
+time();
+stringToNumber(string1);
+numberToString(num1);
+console.log(whatDataType(dataType));
+console.log(addTwoNumber(num1, num2));
+console.log(bothTrue());
+console.log(oneTrue());
+console.log(bothFalse());
+
+function stringToNumber(string1) {
+  const string = parseInt(string1);
+  console.log(typeof(string), string);
+  return
+}
+
+function numberToString(num1) {
+  const num = num1.toString();
+  console.log(typeof(num), num);
+  return
+}
+
+function whatDataType() {
```

Comment:
you need to pass datatype to your function as well. right now your function is not taking in any parameters